Cargo.toml

```diff
@@ -48,5 +48,8 @@ sqlx = {version = "0.6.1", features = ["runtime-tokio-native-tls", "uuid", "json
 # To stream over sqlx results
 futures = "0.3"
 
+prima-tracing = "0.5.*"
```

Comment:
I think we shouldn't depend on prima-tracing. It seems we are not using it and imho it should be up to the users of the library to choose the "tracing implementation" they prefer.

Reply:
I agree on not depending on specific tracing implementations, but would `tracing` work even without `prima-tracing`? Even better: what happens if someone doesn't import `prima-tracing` as a dependency but uses `esrs`?

Reply:
Tracing needs a handler to be set by the application, prima-tracing is just a few helper functions for initializing the tracing event handler. Everything will work fine without prima-tracing

Reply:
As far as I know, if someone doesn't enable tracing, these trace calls become no-ops (if nobody sets up a subscriber, there's nowhere to send the data) - in this case, `tracing` would work fine but do nothing. We could gate these traces behind a feature, but honestly just "if you've enabled tracing you'll get some spans etc for esrs" seems like a fine state of affairs.

@Fs00 you're right, it doesn't depend on `prima-tracing`, since we don't configure the subscriber, which is mainly what that's for. I'll pull it out.